src/model/nodes/person.js

```diff
@@ -36,6 +37,19 @@ PersonNode.prototype.setTitle = function (person) {
 };
 
 /**
+ * Gets email for node
+ * @param {Object} person - object data for person
+ * @returns {PersonNode}
+ */
+PersonNode.prototype.getEmail = function (person) {
+    this.email = utility.getLanguages().reduce(function (prev, lang) {
+        prev[lang] = util.format('%s', person[lang]['email']);
```

Comment:
what is the reason to use `util.format()` here?

as for codestyle, I prefer to write property lookup in dot notation (`person[lang].email`).